pkg/util/protoutil/jsonpb_marshal.go

```diff
@@ -133,3 +133,8 @@ func (j *JSONPb) NewEncoder(w io.Writer) gwruntime.Encoder {
 		return errors.Errorf("unexpected type %T does not implement %s", v, typeProtoMessage)
 	})
 }
+
+// Delimiter implements gwruntime.Delimited.
+func (*JSONPb) Delimiter() []byte {
```

Comment:
Oh, we can add the static assertions now, right?

Let's squash these commits while we're at it - no need to do it in two steps.

Reply:
Ah, of course. Will do :).